casa/System/AipsrcValue.h

```diff
@@ -219,22 +206,13 @@ template <class T> class AipsrcValue : public Aipsrc {
 private:
   //# Data
   // The global AipsrcValue object
-  static AipsrcValue myp_p;
-  static std::mutex theirMutex;
+  inline static std::mutex theirMutex;
   // Register list
   // <group>
-  Block<T> tlst;
-  Block<String> ntlst;
+  inline static std::vector<T> tlst;
+  inline static std::vector<String> ntlst;
```

Comment:
Similarly to !1422, I don't see the point of using `inline` member variables. Please explain, or use plain `static` members, without `inline`.

Reply:
Resolved in !1422.